### PR TITLE
Apply 'arrow-return-shorthand' lint rule

### PR DESCRIPTION
**Pull request by andy-ms**

### CODE REVIEW

File: src/server/session.ts [hunk 1]
```diff
@@ -1742,42 +1740,30 @@ namespace ts.server {
                 this.exit();
                 return this.notRequired();
             },
-            [CommandNames.Definition]: (request: protocol.DefinitionRequest) => {
```
Comment (mhegazy) [inline src/server/session.ts]: can we keep these for consistency.. some will need to have the curlies..

File: src/server/session.ts [hunk 2]
```diff
@@ -1690,284 +1707,188 @@ namespace ts.server {
...
+        }
+
+        private noResponse(cb: () => void): HandlerResponse {
+            cb();
+            return { responseRequired: false };
+        }
+
         private handlers = createMapFromTemplate<(request: protocol.Request) => HandlerResponse>({
-            [CommandNames.OpenExternalProject]: (request: protocol.OpenExternalProjectRequest) => {
+            [CommandNames.OpenExternalProject]: (request: protocol.OpenExternalProjectRequest) => this.respondTrue(() => {
```
Comment (mhegazy) [inline src/server/session.ts]: seems a bit weird to add a new dispatch just to make the linter happy..

Comment (sheetalkamat) [reply src/server/session.ts]: +1 It also means we are creating unnecessary function expression per handler..

Comment (mhegazy) [reply src/server/session.ts]: do we even need this lint rule...

**Comment by mhegazy** [comment]
Do not believe this rule is needed. i would rather keep the code the way it is and keep this rule excluded.

**Pull request closed without merging**